### PR TITLE
Added Github release script

### PR DESCRIPTION
**Pull request by aaronwaggener**
Creates releases in Github like [this one](https://github.com/code-dot-org/browser/releases/tag/v1.1.9). Builds for our supported operating systems are attached as Release Assets.

Jira: [STAR-1584](https://codedotorg.atlassian.net/browse/STAR-1584)

image

### CODE REVIEW

File: github-release.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+if (accessToken) {
+  (async() => {
+    const octokit = new Octokit({
+      auth: accessToken,
+    });
+    const release = await createRelease();
+    const releaseId = release.data.id;
+
+    await uploadAssets(releaseId);
+    console.log(`Release drafted. Visit ${release.data.html_url} to publish.`);
```
Comment (madelynkasula) [inline github-release.js:18]: should we automatically publish the release at this step (using `octokit.rest.repos.updateRelease` to set `draft` to false)? 

if there's anything we want the deploying engineer to double-check, i don't think we should automatically publish. and, in that case, we should add those steps to this log (e.g., "make sure ${buildNameSuffixes} have all been uploaded")

File: github-release.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      });
+    }
+
+    function uploadAssets(releaseId) {
+      return new Promise(async(resolve, reject) => {
+        for (let i = 0; i < buildNameSuffixes.length; i++) {
+          let buildName = buildNamePrefix + buildNameSuffixes[i];
+          let buildFile = fs.readFileSync(`./dist/${buildName}`);
+          console.log(`Uploading ${buildName}`);
+          await octokit.rest.repos.uploadReleaseAsset({
```
Comment (madelynkasula) [inline github-release.js:37]: should we log an error and/or `reject` if any asset fails to upload?

File: github-release.js (hunk 1 above)
Comment (aaronwaggener) [reply github-release.js:18]: We want the engineer to add release notes before publishing. I will update the wording here 👍

File: github-release.js (hunk 2 above)
Comment (aaronwaggener) [reply github-release.js:37]: Good catch. Yes, we definitely should. I will add logic to handle failed uploads.

Comment (aaronwaggener) [reply github-release.js]: @madelynkasula I added error messaging [here](https://github.com/code-dot-org/browser/pull/96/files#diff-2cd718574419de1e5815325176d6e423dedf36fc4ffbe14998aa6d173ec7f6b6R44-R46). I am not rejecting the promise as I think we should continue attempting to upload the remaining assets.

File: github-release.js (hunk 1 above)
Comment (aaronwaggener) [reply github-release.js]: ...updated to include a nudge to add release notes

**Review by madelynkasula — APPROVED** [review]
this looks great -- thank you for automating this task!

**Pull request merged**